recipes/termcap/all/conanfile.py

```diff
@@ -72,9 +77,16 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
 
+    @property
+    def _termcap_path(self):
+        return os.path.join(self.package_folder, "bin", "etc", "termcap")
```

Comment:
`bin/etc`? Should it be just `etc` (https://cmake.org/cmake/help/v3.4/module/GNUInstallDirs.html)?

Reply:
Yes, but the hooks don't allow a root directory other then `bin`, `lib` and `licenses`.

Reply:
👍 Understood. I'm not a consumer of this library, so I cannot say if having this file in this unexpected directory is a problem or not, I see you can find it using the envvar `TERMCAP`. No idea if this will be a problem.

Anyway, probably it's better to package the file like this and, if someone complains, we can always modify the hook to allow this file in its proper place.

Thanks!